src/Styleguide/Components/ArtistInsightsModal.tsx

```diff
@@ -0,0 +1,73 @@
+import { Link, Sans, Separator, Serif } from "@artsy/palette"
+
+import { Modal } from "Components/Modal/Modal"
+import React from "react"
+
+export class ArtistInsightsModal extends React.Component {
+  state = {
+    showModal: false,
+  }
+
+  render() {
+    return (
+      <>
+        <Modal
+          onClose={() => this.setState({ showModal: false })}
+          show={this.state.showModal}
+          style={{
+            maxHeight: 540,
+            overflowX: "scroll",
+          }}
+        >
+          <Serif size="5" textAlign="center" weight="semibold">
+            Artist Insights
+          </Serif>
+          <Serif
+            size="2"
+            mt={1}
+            textAlign="center"
+            color="black60"
+            lineHeight="24px"
+          >
+            Artist Insights exists to help art buyers understand the cultural
+            significance of artists on Artsy. We aim to contextualize artists
```

Comment:
We should have someone (maybe from Editorial?) review the copy and make sure the grammar/language is correct.